core/src/test/java/com/radixdlt/rev2/nofaucet/CallPreviewWithoutFaucetTest.java

```diff
@@ -0,0 +1,185 @@
+ * permissions under this License.
+ */
+
+package com.radixdlt.rev2.nofaucet;
+
+import static com.radixdlt.environment.deterministic.network.MessageSelector.firstSelector;
+import static com.radixdlt.lang.Tuple.tuple;
+import static org.junit.Assert.assertEquals;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Module;
+import com.radixdlt.addressing.Addressing;
+import com.radixdlt.api.core.generated.api.TransactionApi;
+import com.radixdlt.api.core.generated.client.ApiException;
+import com.radixdlt.api.core.generated.models.*;
+import com.radixdlt.crypto.ECKeyPair;
+import com.radixdlt.environment.deterministic.network.MessageMutator;
+import com.radixdlt.genesis.*;
+import com.radixdlt.harness.deterministic.DeterministicTest;
+import com.radixdlt.harness.deterministic.PhysicalNodeConfig;
+import com.radixdlt.identifiers.Address;
+import com.radixdlt.modules.FunctionalRadixNodeModule;
+import com.radixdlt.modules.StateComputerConfig;
+import com.radixdlt.networks.Network;
+import com.radixdlt.rev2.Decimal;
+import com.radixdlt.rev2.ScryptoConstants;
+import com.radixdlt.rev2.protocol.ProtocolUpdateTestUtils;
+import com.radixdlt.utils.UInt32;
+import com.radixdlt.utils.UInt64;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public final class CallPreviewWithoutFaucetTest {
+  @Rule public TemporaryFolder folder = new TemporaryFolder();
+
+  private DeterministicTest createTest(Module... extraModules) {
+    final var validatorKey = ECKeyPair.fromSeed(new byte[] {0x02}).getPublicKey();
+
+    var genesisData =
+        new GenesisData(
+            UInt64.fromNonNegativeLong(1L),
+            0,
+            GenesisConsensusManagerConfig.testingDefaultEmpty(),
+            ImmutableList.of(
+                new GenesisDataChunk.Validators(
+                    ImmutableList.of(
+                        new GenesisValidator(
+                            validatorKey,
+                            true,
+                            true,
+                            Decimal.ZERO,
+                            ImmutableList.of(),
+                            Address.virtualAccountAddress(validatorKey)))),
+                new GenesisDataChunk.Stakes(
+                    ImmutableList.of(Address.virtualAccountAddress(validatorKey)),
+                    ImmutableList.of(
+                        tuple(
+                            validatorKey,
+                            ImmutableList.of(
+                                new GenesisStakeAllocation(
+                                    UInt32.fromNonNegativeInt(0), Decimal.ONE)))))),
+            Decimal.ofNonNegative(0L),
+            ImmutableList.of());
+
+    return DeterministicTest.builder()
+        .addPhysicalNodes(PhysicalNodeConfig.createBatch(1, true))
+        .messageSelector(firstSelector())
+        .messageMutator(MessageMutator.dropTimeouts())
```

Comment:
Why are we dropping timeouts?

Reply:
These are leftovers from the test I've used as a base.